_episodes/23-object-oriented.md

```diff
@@ -516,22 +512,19 @@ If we want to extend the previous example to also manage people who aren't acade
 But `Person` will share some data and behaviour with `Academic` - in this case both have a name and show that name when you print them.
 Since we expect all academics to be people (hopefully!), it makes sense to implement the behaviour in `Person` and then reuse it in `Academic`.
```

Comment:
Feels like the text "If we want to extend the previous example to also manage people who aren’t academics we can add another class Person. But Person will share some data and behaviour with Academic - in this case both have a name and show that name when you print them. Since we expect all academics to be people (hopefully!), it makes sense to implement the behaviour in Person and then reuse it in Academic." remained from the previous version and needs to be rewritten to replace Academic with Patient.